README.md

```diff
@@ -8,32 +8,35 @@ Configure this ML Model service as a modular resource on your machine with a [Je
 
 ## Requirements
 
-A NVIDIA Jetson Orin board with the following installed:
+A NVIDIA Jetson Orin board or other machine with an NVIDIA GPU with the following installed:
 
-1. [Jetpack 5](https://developer.nvidia.com/embedded/jetpack)
+1. [CUDA](https://developer.nvidia.com/cuda-toolkit), [Jetpack 5, or Jetpack 6](https://developer.nvidia.com/embedded/jetpack)
 2. NVIDIA Container Runtime
 
-Run the following to install `nvidia-jetpack` and `nvidia-container` on your robot's computer:
+On a Jetson Orin machine, run the following to install `nvidia-jetpack` and `nvidia-container` on your robot's computer:
```

Comment:
I believe you need `nvidia-container` even on a non-JP machine.

Reply:
Good point, thanks!